pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3139,7 +3139,7 @@ func buildListener(port v1.ServicePort, annotations map[string]string, sslPorts
 	listener.InstancePort = &instancePort
 	listener.LoadBalancerPort = &loadBalancerPort
```

Comment:
I don't think LoadBalancerPort is changed from the service port, so I don't think a service that is configured in k8s with port 80 is still going to listen on port 80?  Unless one of the other options overrides things?